parameter_files/fates_params_default.cdl

```diff
@@ -11,6 +11,7 @@ dimensions:
 	fates_pft = 12 ;
 	fates_prt_organs = 4 ;
 	fates_string_length = 60 ;
+	fates_hlm_pftno = 14 ;
```

Comment:
Can one add comments to this part of the parameter file? If so, I think it would be useful to make clear what this is, lest people are confused bout  its meaning  in relation to the FATES pft number above

Reply:
I'm not exactly sure how one would go about associating metadata with a dimension size, or to add a comment that would preserve its location next to the dimension.  It might be possible though.